wordpress-mobile/gutenberg-mobile
# Search Block 1: Add block to inserter

Related to #3073. Updates the gutenberg reference to point to the branch that adds the search block to the inserter (dev mode only). This is just one of many steps for this block. The block added to the editor is just a skeleton block with most functionality stripped away (only the search label is shown). Subsequent PR's will continue to add functionality. 

### Gutenberg PR
https://github.com/WordPress/gutenberg/pull/28983

### To test:
1. From the editor, click to open the block inserter. Verify the Search block is displayed.
2. Click on the search block to add it to the post. Verify no errors and the search block shell has successfully been added. 

PR submission checklist:
